services/feeds.js

```diff
@@ -0,0 +1,160 @@
+var request = require('superagent');
+
+/**
+ * @constructor
+ */
+
+function Feeds(defaults) {
+
+	defaults = defaults || {};
+
+	/* set default format or lang for all requests */
+
+	/* we default to json since it's a js-sdk */
+	this.format = defaults.format || 'json';
+	this.lang = defaults.lang;
```

Comment:
Is there a reason to make these properties? Just set them on the request instance, then return it and people can override them in userland if they want. That's the pattern we use in request.js for the REST API.